### ISSUE TITLE
Questions about the Paper

### ISSUE THREAD

**Issue by Shayne-Ada** [chintu619/Joint-ASR-AAC#3]
Hello, I have two questions regarding the papers:

1. In Section 3, it is mentioned that if x2 is shorter than x1, x1 will be randomly truncated. Will this lead to a loss of audio content and result in misalignment with the transcribed text?

2. Also in Section 3, it is mentioned that when constructing captions, they are all generated in the form of "a person is speaking" and all training data contains human voices. Will this cause the model to output the above content even in non-speech segments?
